recipes/maddy/all/test_package/example.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ *  This example was copied from
+ * https://github.com/progsource/maddy/blob/master/README.md
+ */
+
+#include <cstdlib>
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include <maddy/parser.h>
+
+int main() {
+  std::stringstream markdownInput(
+      "#Hello world from maddy!\nVisit "
```

Comment:
Looks like there must be a space after the hashtag.
```suggestion
      "# Hello world from maddy!\nVisit "
```